utils/emailer.js

```diff
@@ -0,0 +1,35 @@
+const config = require('config');
+
+const emailConfig = config.get('development.email');
+
+const httpLogger = require('./logger');
+const nodemailer = require('nodemailer');
```

Comment:
Move package requires before local requires. (This is an eslint rule, but in babel, which is why it doesn't show up here.)